src/pyfme/utils/change_euler_quaternion.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jan 12 22:34:43 2016
+
+@author: Juan
+"""
+
+from math import atan2, asin, cos, sin
+import numpy as np
+
+
+def quatern2euler(quaternion):
+    '''Given a quaternion, the euler_angles vector is returned.
+
+    Parameters
+    ----------
+    quaternion : array_like
+        1x4 vector with the four elements of the quaternion:
+        [q_0, q_1, q_2, q_3]
+
+    Returns
+    -------
+    euler_angles : array_like
+        1x3 array with the euler angles: [theta, phi, psi]    (rad)
+
+    References
+    ----------
+    .. [1] "Modeling and Simulation of Aerospace Vehicle Dynamics" (Aiaa\
+        Education Series) Peter H. Ziepfel
+    '''
+    check_unitnorm(quaternion)
+
+    q_0, q_1, q_2, q_3 = quaternion
+
+    check_unitnorm(quaternion)
```

Comment:
Maybe this is too much if it has to be evaluated each time step. But I think we'll have time to get rid of it later